packages/trader/src/sass/app/modules/trading.scss

```
@@ -146,13 +146,27 @@
         }
         .dc-datepicker {
             .dc-input {
-                border: 1px solid var(--fill-normal);
                 background-color: var(--general-main-1);
+                border-radius: $BORDER_RADIUS;
                 margin-bottom: 0;
 
                 &__field {
                     height: 3.2rem;
                 }
+                &__container {
+                    border: none;
+
+                    &:hover:not(.dc-input--disabled) {
+                        border: 1px solid var(--border-hover);
+                    }
+                    &:focus-within {
+                        border: 1px solid var(--brand-secondary);
+
+                        &:hover {
+                            border: 1px solid var(--brand-secondary);
```

```suggestion
                            border-color: var(--brand-secondary);
```
Since only the color change, you can just declare the `border-color` property

---

Hi @yashim-deriv ! But `&__container` has `border: none;` 
So in case if I add just `border-color` on hover/focus to element without border, then it wouldn't be visible. Please, correct me if I'm mistaken.

---

shouldn't we use `--border-hover` here instead of `--brand-secondary`?
Also, can we simplify the nesting of these pseudo classes?

---

Hi @balakrishna-deriv! 
`--brand-secondary` is green color, which is used for active (focused) elements. So if the element is in focus, it should have green color of border, not grey (`--border-hover`). Same if the element is in focus + we hover it ---> border color should be green

---

Expected border color: 
Default: No border
Hover: Grey border
Active: Green border